doc/memos/rdm-draft-petry-riot-design-goals.md

```diff
@@ -0,0 +1,218 @@
+
+ - RDM: 1
+ - Title: RIOT design goals
+ - Author: Daniel Petry  
+ - Date: October 2018
+
+## Abstract
+
+This memo describes RIOT's high level design goals. The aim is to reflect
+community consensus on these goals and thereby guide technical discussion around
+specific contributions.
+
+## Status
+
+This document is a product of the community of RIOT maintainers, and aims to 
```

Comment:
Since the document is not a consensus document yet, the first sentence should be remove for now (added when/if the document is about to be published and assigned an RDM number etc.)

Reply:
@danpetry can you modify asap to disambiguate?